go.mod

```diff
@@ -4,23 +4,23 @@ go 1.14
 
 require (
 	github.com/aryann/difflib v0.0.0-20170710044230-e206f873d14a
-	github.com/evanphx/json-patch v4.5.0+incompatible
+	github.com/evanphx/json-patch v4.9.0+incompatible
 	github.com/ghodss/yaml v1.0.0
-	github.com/google/go-cmp v0.5.1
-	github.com/google/go-containerregistry v0.1.1
+	github.com/google/go-cmp v0.5.5
+	github.com/google/go-containerregistry v0.4.1
 	github.com/mgutz/ansi v0.0.0-20170206155736-9520e82c474b
-	github.com/pivotal/kpack v0.1.5-0.20210107191603-fa0a973272f5
+	github.com/pivotal/kpack v0.2.3-0.20210430201452-ed7fe5056666
 	github.com/pkg/errors v0.9.1
 	github.com/sclevine/spec v1.4.0
 	github.com/spf13/cobra v1.0.0
 	github.com/stretchr/testify v1.6.1
-	golang.org/x/crypto v0.0.0-20200510223506-06a226fb4e37
-	golang.org/x/sync v0.0.0-20200317015054-43a5402ce75a
-	k8s.io/api v0.17.6
-	k8s.io/apimachinery v0.17.6
+	golang.org/x/crypto v0.0.0-20201016220609-9e8e0b390897
+	golang.org/x/sync v0.0.0-20210220032951-036812b2e83c
+	k8s.io/api v0.19.7
+	k8s.io/apimachinery v0.19.7
 	k8s.io/client-go v11.0.1-0.20190805182717-6502b5e7b1b5+incompatible
-	knative.dev/pkg v0.0.0-20200702222342-ea4d6e985ba0
+	knative.dev/pkg v0.0.0-20210331065221-952fdd90dbb0
 	sigs.k8s.io/yaml v1.2.0
 )
 
-replace k8s.io/client-go => k8s.io/client-go v0.17.5
+replace k8s.io/client-go => k8s.io/client-go v0.19.7
```

Comment:
this seems to imply we don't need the replace anymore: https://github.com/pivotal/kpack/pull/681/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R54

Reply:
seeing this failure after removing it and running go mod tidy 
```
go: finding module for package k8s.io/api/auditregistration/v1alpha1
github.com/pivotal/build-service-cli/pkg/testhelpers imports
        k8s.io/client-go/kubernetes/fake imports
        k8s.io/api/auditregistration/v1alpha1: module k8s.io/api@latest found (v0.21.0), but does not contain package k8s.io/api/auditregistration/v1alpha1
```

Reply:
was able to remove it by bumping to the 0.19.7 release of client-go